main/src/ca/uwaterloo/flix/tools/pkg/FlixPackageManager.scala

```diff
@@ -17,25 +17,43 @@ package ca.uwaterloo.flix.tools.pkg
 
 import ca.uwaterloo.flix.api.Bootstrap
 import ca.uwaterloo.flix.tools.pkg.Dependency.FlixDependency
+import ca.uwaterloo.flix.tools.pkg.FlixPackageManager.downloadFromGitHub
 import ca.uwaterloo.flix.tools.pkg.github.GitHub
-import ca.uwaterloo.flix.util.Result.{Err, Ok, ToOk}
+import ca.uwaterloo.flix.util.Result.{Err, Ok, ToOk, traverse}
 import ca.uwaterloo.flix.util.Result
+import org.apache.commons.io.FileUtils
 
 import java.io.{IOException, PrintStream}
 import java.nio.file.{Files, Path, StandardCopyOption}
 import scala.util.Using
 
 object FlixPackageManager {
 
+  //All Maven dependencies found transitively from Flix dependencies
+  val mavenSet = collection.mutable.Set.empty[String]
```

Comment:
Since I am finding Maven dependencies I need a way to pass these from the FlixPackageManager to the MavenPackageManager. I could return the list of Maven dependencies discovered as a tuple with the list of paths to the Flix dependencies, but I thought that was very ugly, so I saved the set of Maven dependencies in a public field instead. Is that alright or do you prefer returning a tuple?

Reply:
We can never use mutable state because there can be multiple instances of the compiler running.

Reply:
We will have to extract them from the parsed Flix toml files.

Reply:
I would also not be returning strings. We should be returning the Manifest data structure and building on that.

Reply:
A list of manifests would seem most reasonable to me.